plugin/impl.go

```diff
@@ -294,3 +320,57 @@ func getServerWithDefaults(server string, host string, protocol string) string {
 
 	return fmt.Sprintf("%s://%s", protocol, host)
 }
+
+func blockUntilBuildIsFinished(p *Plugin, client drone.Client, namespace, name string, buildNumber int) error {
+	fmt.Printf("\nblocking until triggered build is finished (canceling this build will cancel the downstream build too)\n")
+
+	timeout := time.After(p.settings.BlockTimeout)
+
+	//lint:ignore SA1015 refactor later
+	tick := time.Tick(10 * time.Second)
+
+	// listen for SIGINT and SIGTERM to cancel downstream build when our build is cancelled
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+	defer close(sigs)
+
+	for {
+		select {
+		case <-sigs:
```

Comment:
if the pipeline is cancelled Drone will send a SIGKILL to the container which means the container will be immediately killed and this would not execute.

Reply:
Hm .. it turned out to be convenient for local testing at least.

Would you agree to keep it when I add a comment and remove texts that claim that this works in drone?

Reply:
There's probably no way at all to handle this in drone?

Reply:
Commented it properly now.

Cancellation hooks are not a thing in drone, right?